lib/controllers/users.js

```diff
@@ -0,0 +1,7 @@
+const { Router } = require('express');
+const authenticate = require('../middleware/authenticate');
```

Comment:
Be wary of adding lines of code that will be necessary later, but aren't in use right now. This unused variable would cause CI to fail if you pushed your commit to an open PR. Also, if you hadn't created the authenticate file, your test would fail on this line when it isn't able to find the required file to import.